Src/Couchbase.Linq/Execution/IBucketQueryExecutor.cs

```diff
@@ -43,15 +44,17 @@ internal interface IBucketQueryExecutor : IQueryExecutor
         /// </summary>
         /// <typeparam name="T">Type returned by the query.</typeparam>
         /// <param name="queryRequest">Request to execute.</param>
+        /// <param name="cancellationToken">Cancellation token.</param>
         /// <returns>Task which contains a list of objects returned by the request when complete.</returns>
-        Task<IEnumerable<T>> ExecuteCollectionAsync<T>(LinqQueryRequest queryRequest);
+        Task<IEnumerable<T>> ExecuteCollectionAsync<T>(LinqQueryRequest queryRequest, CancellationToken cancellationToken);
```

Comment:
Changing the interface definition here can cause semver problems with dependent projects. Instead, we've found it better to provide an overload with the additional parameters with implementations redirected to the new implementation with defaults.

Reply:
This is an internal interface, so I was able to change it freely.

Reply:
Ah, missed it is internal as the file was truncated for review.